README.md

```diff
@@ -124,23 +55,23 @@ In the terminal, create an `index.html` file:
 $ touch index.html
 ```
 
-Open this file in your text editor so we can add some basic HTML content. In the
-file, we'll start with the minimum — a `DOCTYPE` tag indicating HTML and
-an `html` tag containing `head` and `body` tags:
+Open this file in VS Code so we can add some basic HTML content. In the file,
+we'll start with the minimum — a `DOCTYPE` tag indicating HTML and an `html` tag
+containing `head` and `body` tags:
 
 ```html
 <!DOCTYPE html>
 <html lang="en">
-  <head> </head>
+  <head></head>
   <body></body>
 </html>
 ```
 
-Let's give this HTML a `title` tag and place it inside `head`:
+Next, let's give this HTML a `title` tag and place it inside `head`:
 
 ```html
 <head>
-  <title>My Website</title>
+  <title>My Personal Website</title>
```

Comment:
The `<title>` in the snippet on line 94 needs to be updated to match this as well

Reply:
👍  Thanks Ian!